.gitignore

```diff
@@ -113,4 +113,5 @@ tutorial/example.db
 docs/_build/
 docs/source/reference/generated/
 docs/source/reference/multi_objective/generated/
+docs/source/reference/visualization/generated/
```

Comment:
I think this directory can be removed when we execute `make clean` in the `doc` directory. Could you update `docs/Makefile` as well?